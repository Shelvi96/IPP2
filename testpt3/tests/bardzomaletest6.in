NEW ID

30>48:758918
616>684:028039
4:2;224;>;4919953:
844068779:>:660697
8;1>061
4907993:7>929;7
553;2:12>3:29445
91::805898>078604
76;98603>4;1986;620
3>0427
0;996388>283
42;13501;>52697
800>9964:
@06;2:653124228086936:76:88816
@9:19147:781;:302;60752
@3
@3;88535892111;890087427:6
@780;87:938094935254369724
@531;9902881:
@962:
@31;6:89554;9781
@0254850:42;2495724110
@:9:
@7520266;:39251443810835989
@:7246
@::24416;:04350;
@5;11831:7499
