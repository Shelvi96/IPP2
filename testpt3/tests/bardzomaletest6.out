2702446210547712
75420314
0
130326302592
28961480559
0
0
0
110001
0
1563915631104
0
1
0
